src/protocols/aeternity/composables/aeTokenSales.ts

```diff
@@ -0,0 +1,84 @@
+import { computed, ref } from 'vue';
+
+import { ITokenSale } from '@/types';
+import { PROTOCOLS, STORAGE_KEYS } from '@/constants';
+import { fetchAllPages, fetchJson } from '@/utils';
+import { useNetworks, useStorageRef } from '@/composables';
+import { useAeNetworkSettings } from './aeNetworkSettings';
+
+let composableInitialized = false;
+
+const areTokenSalesInitialized = ref(false);
+
+/**
+ * List of all token sales available
+ */
+const tokenSales = useStorageRef<ITokenSale[]>(
```

Comment:
I don't see any reason to use `useStorageRef`. Tokens are being fetched on every wallet initialization.